### PR TITLE
Feature/error handler

### PR DESCRIPTION
**Pull request by uino59**
Delegates error handling to its own module which can be utilised by future independent features. In raidpackage, ensure user can't input quantities over the maximum allowed.

### CODE REVIEW

File: error_handler.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        self.error_messages = {}
+
+    async def send_usr_error_message(self, user: int, message: str):
+        error_msg = await user.send(message)
+        if user.id in self.error_messages:
+            self.error_messages[user.id].append(error_msg)
+        else:
+            self.error_messages[user.id] = [error_msg]
+
+        print(self.error_messages)
```
Comment (msadowskigraduate) [inline error_handler.py:15]: Mister

File: error_handler.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import discord
+import asyncio
+
+class Error_handler():
+    def __init__(self):
+        self.error_messages = {}
```
Comment (msadowskigraduate) [inline error_handler.py:6]: should be a private variable -> use OOP encapsulation

File: error_handler.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            self.error_messages[user.id].append(error_msg)
+        else:
+            self.error_messages[user.id] = [error_msg]
+
+        print(self.error_messages)
+
+    async def delete_usr_error_messages(self, user:int):
+        if user.id in self.error_messages:
+            del self.error_messages[user.id]
+            print(self.error_messages)
```
Comment (msadowskigraduate) [inline error_handler.py:20]: no debug printing in production code

File: raidpackage.py [hunk 4]
```diff
@@ -1,7 +1,10 @@
 import asyncio
+from error_handler import Error_handler
+from logging import error
 import discord
 import item_definitions
 
+
```
Comment (msadowskigraduate) [inline raidpackage.py:7]: unnecessary extra newline

File: raidpackage.py [hunk 5]
```diff
@@ -123,10 +128,9 @@ async def __cancel_order(self, msg, usr_id):
     def __gracefully_complete_order(self, order: Order):
         del self.__responding_players[order.id]
 
-    async def __process_user_quantity_input(self, client, usr, item, qtyMax, usr_id):
-        __error_messages = []
+    async def __process_user_quantity_input(self, client, usr, item, qtyMax, usr_id, order: Order):
```
Comment (msadowskigraduate) [inline raidpackage.py:131]: Since we are passing `Order` we no longer need `qtyMax` and `usr` and `usr_id`

**Review by msadowskigraduate — CHANGES_REQUESTED** [review]
Goooood job

File: error_handler.py (hunk 1 above)
Comment (uino59) [reply error_handler.py:15]: Removed 324368a

File: error_handler.py (hunk 2 above)
Comment (uino59) [reply error_handler.py:6]: Privatised a94ae30

File: error_handler.py (hunk 3 above)
Comment (uino59) [reply error_handler.py:20]: Removed 324368a

File: raidpackage.py (hunk 4 above)
Comment (uino59) [reply raidpackage.py:7]: removed

File: raidpackage.py (hunk 5 above)
Comment (uino59) [reply raidpackage.py]: Fixed 34da2ec

**Pull request merged**